### ISSUE TITLE
Post shown isn't always the correct one in background tabs left open over a long time

### ISSUE THREAD

**Issue by 7heo** [LemmyNet/lemmy-ui#1094]
### Issue Summary

After leaving a background tab open, and coming back to it, the displayed post is sometimes different than the one the URL of the tab points to. For example, [How the web became unreadable](https://lemmy.ml/post/1149760) became [Lemmy’s iOS Jailbreak community is dead so i made a new one.](https://lemmy.ml/post/1155008), and everything went back to normal after a refresh.

### Steps to Reproduce

1. Post a new post on Lemmy
2. Keep it open in a background tab
3. After a long while (hours?), come back to it
4. Check the displayed post
5. Refresh the post
6. Compare the displayed post

### Technical details

I am using Firefox 102.10.0esr from Kali x86_64.

I suppose this bug is related to #1093.

**Comment by 7heo** [comment]
@dessalines could you then please create two "on-hold" labels for issues and assign them to #1093 and the current issue? Also it would be good to have a milestone for 'http-rework' and one for what comes after (if you have any idea, otherwise 'post-http-rework') and assign that milestone to both issues. So people know what's up at a glance ;)

**Comment by dessalines** [comment]
Sure. Once I get some time, I'll go through the lemmy-ui and jerboa repos, and organize them, and try to create more tags. Bit overloaded recently obviously.

**Comment by 7heo** [comment]
Yeah I can imagine. My offer still stands if you feel ok trusting an absolute stranger to organize your issues... ;)

**Comment by Nutomic** [comment]
Websockets are removed so hopefully this is fixed now.
